src/clients/lan_auto.rb

```diff
@@ -159,7 +160,13 @@ def main
         @ret = deep_copy(@autoyast)
       elsif @func == "Write"
         @progress_orig = Progress.set(false)
-        @ret = Lan.WriteOnly
+
+        @ret = LanUdevAuto.Write
+        Builtins.y2error("Writing udev rules failed") if !@ret
+
+        @ret = Lan.WriteOnly && @ret
```

Comment:
so in this usage it means try write Lan even if LanUdevAuto failed? does it make sense? I think at least comment why it is possible should be here.